plans/integration/resolve-dependency/vm_centos8.fmf

```diff
@@ -0,0 +1,15 @@
+discover+:
+    filter+:
+        - 'tag: centos8'
```

Comment:
All the dependency issues we were hitting were on CentOS Linux 7. So you can drop the CentOS 8.